### ISSUE TITLE
Convolution2D with border_mode='same' incorrect

### ISSUE THREAD

**Issue by torsten-wilhelm** [pplonski/keras2cpp#17]
The Convolution2D layer with border_mode='same' does not give the same results as the Keras/Theano version (border_mode='valid' is ok in my test).

**Comment by pplonski** [comment]
Can you show your keras.json file or Keras config that you used? what dim ordering have you used?

**Comment by torsten-wilhelm** [comment]
I use theano ordering.

{
    "image_dim_ordering": "th", 
    "epsilon": 1e-07, 
    "floatx": "float32", 
    "backend": "theano"
}

**Comment by pplonski** [comment]
Do you still have an issue?
